src/sas/qtgui/Plotting/Slicers/SectorSlicer.py

```diff
@@ -269,6 +272,7 @@ def setParams(self, params):
             params["Delta_Phi [deg]"] = MIN_PHI
 
         self.nbins = int(params["nbins"])
+        self.fold =params["fold"]
```

Comment:
Assuming `params` is a dictionary (no typing in the method signature...), this should probably be a `get`, with some default value, and bool coercion. I'm not sure if params is set from an external source (like save states), but existing methods might not have the `fold` key, resulting in a `KeyError`.

Reply:
I think we should keep this as is. `params` is how the slicer parameters that you set in the slicer editor widget (`QTableView`) are passed to the slicer.  This was set up by @rozyczko years ago.  The magic (as far as I'm concerned even after looking at it for a while) is in `SlicerParameters.py` and `SlicerModel.py`.
as stated in `SlicerParameter`
```python
class SlicerParameters(QtWidgets.QDialog, Ui_SlicerParametersUI):
    """
    Interaction between the QTableView and the underlying model,
    passed from a slicer instance.
    """
Unless I completely missed the point being made?

```